test/box-luatest/func_test.lua

```diff
@@ -120,3 +120,12 @@ g.test_gh_7822_vfunc_format = function()
         t.assert_equals(box.space._vfunc:format(), box.space._func:format())
     end)
 end
+
+g.test_ghs_82_tuple_field_by_path = function()
```

Comment:
Why do you think we should add this test case to this file? They look completely unrelated.

BTW I think that it's okay to commit this fix without a test, because testing internal functions isn't really necessary.

Reply:
I decided it because of the test-file name.

Dropped test.